### PR TITLE
Organization ui

### PR DESCRIPTION
**Pull request by Macroz**
Implements the first parts of the UI i.e. the list with state actions of #2039 

I will add tests to the these admin screens once the whole ticket is done.

orgs

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] consider adding screenshots for ease of review

## API
- [x] API is documented and shows up in Swagger UI
- [x] API is backwards compatible or completely new

## Documentation
- [x] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: src/cljs/rems/administration/organizations.cljs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+   (str "/administration/organizations/" organization-id)
+   (text :t.administration/view)])
+
+(rf/reg-sub
+ ::organizations-table-rows
+ (fn [_ _]
+   [(rf/subscribe [::organizations])
+    (rf/subscribe [:language])])
+ (fn [[organizations language] _]
+   (map (fn [organization]
```
Comment (opqdonut) [inline src/cljs/rems/administration/organizations.cljs:78]: for a big `map` like this, I think `for` reads better

File: src/cljs/rems/administration/status_flags.cljs [hunk 2]
```diff
@@ -11,46 +11,51 @@
...
    {:type :button
-    :on-click #(on-change (assoc item :enabled true) [:span [text :t.administration/enable]
-                                                      " \""  [get-localized-title-for-anything item] "\""])}
+    :on-click #(on-change (assoc item :enabled true)
+                          [:span [text :t.administration/enable]
+                           " \""  [get-localized-title-for-anything item] "\""])}
    (text :t.administration/enable)])
 
-; TODO consider naming enabled-toggle-button
+                                        ; TODO consider naming enabled-toggle-button
```
Comment (opqdonut) [inline src/cljs/rems/administration/status_flags.cljs:33]: maybe just add the second `;`?

File: src/cljs/rems/administration/status_flags.cljs [hunk 3]
```diff
@@ -11,46 +11,51 @@
...
 (defn enabled-toggle [item on-change]
-  (if (:enabled item)
+  (if (or (:enabled item) (:enabled item))
```
Comment (opqdonut) [inline src/cljs/rems/administration/status_flags.cljs:35]: that... sounds a bit redundant :) leftover from the organization/enabled refactoring?

File: src/cljs/rems/administration/status_flags.cljs [hunk 4]
```diff
@@ -11,46 +11,51 @@
...
-                                                        " \"" [get-localized-title-for-anything item] "\""])}
+    :on-click #(on-change (assoc item :archived false)
+                          [:span [text :t.administration/unarchive]
+                           " \"" [get-localized-title-for-anything item] "\""])}
    (text :t.administration/unarchive)])
 
 ;; TODO consider naming archived-toggle-button
 (defn archived-toggle [item on-change]
-  (if (:archived item)
+  (if (or (:archived item) (:archived item))
```
Comment (opqdonut) [inline src/cljs/rems/administration/status_flags.cljs:58]: ditto

File: src/cljs/rems/administration/organizations.cljs (hunk 1 above)
Comment (Macroz) [reply src/cljs/rems/administration/organizations.cljs:78]: I think this is just leftover copypasta from earlier instances of us doing this mapping. I'll change it.

File: src/cljs/rems/administration/status_flags.cljs (hunk 2 above)
Comment (Macroz) [reply src/cljs/rems/administration/status_flags.cljs:33]: Yeah makes sense.

File: src/cljs/rems/administration/status_flags.cljs (hunk 3 above)
Comment (Macroz) [reply src/cljs/rems/administration/status_flags.cljs:35]: Yeah I thought I got them all but missed some :(

**Pull request merged**